src/toil/test/cwl/cwlTest.py

```diff
@@ -133,8 +137,11 @@ def test_run_conformance(self):
             shutil.move("common-workflow-language-%s" % testhash, cwlSpec)
             os.remove("spec.zip")
         try:
-            subprocess.check_output(["bash", "run_test.sh", "RUNNER=toil-cwl-runner", "DRAFT=v1.0"],
-                                    cwd=cwlSpec, stderr=subprocess.STDOUT)
+            cmd = ["bash", "run_test.sh", "RUNNER=toil-cwl-runner",
+                   "DRAFT=v1.0", "-j4"]
```

Comment:
I think `"-j4"` should be `"-j", "4"` and that might be causing the test to fail.